app/client/user/users.controller.js

```diff
@@ -96,6 +109,83 @@ angular.module('app.user')
         return vm.selection.filter(function(elem){ return elem; }).length;
       };
 
+
+      function getRole (user) {
+
+        if ( ! user.lomis_stock || ( ! user.lomis_stock.mobile || ! user.lomis_stock.dashboard) ||   ( _.isEmpty(user.lomis_stock.mobile) && _.isEmpty(user.lomis_stock.dashboard))) { return 'None'; }
+        else if (_.isEmpty(user.lomis_stock.mobile)) { return 'Dashboard'; }
+        else { return 'Mobile'; }
+      };
+
+      function getAdminLevel (user) {
+
+
+        switch ( getRole (user) ) {
+          case 'Mobile':
```

Comment:
There was a need to include details of each user's role on the users list. Due to time constraint, an update was made which increased the coupling between the users and lomis_stock role (as seen above). The next PR would include a commit which would de-couple it.